app/src/pages/explorer/Gallery.vue

```diff
@@ -0,0 +1,182 @@
+            >
+              <md-icon>bookmark_border</md-icon>
+            </div>
+            <!-- TODO show only for admin -->
+            <div @click.prevent="deleteChart(result)">
+              <md-icon>delete_outline</md-icon>
+            </div>
+          </div>
+          <router-link :to="`/explorer/chart/view/${result.id}`">
+            <md-card-media-cover
+              md-solid
+            >
+              <md-card-media md-ratio="4:3">
+                <img
+                  :src="getViewUrl(result.thumbnail)"
+                  :alt="result.label"
+                  v-if="result.thumbnail"
+                >
+                <img
+                  :src="defaultImg"
+                  :alt="result.label"
+                  v-else
+                >
+              </md-card-media>
+              <md-card-area class="u_gridbg">
+                <md-card-header class="u_show_hide">
+                  <span class="md-subheading">
+                    <strong>{{ result.label }}</strong>
+                  </span>
+                  <span class="md-body-1">{{ reduceDescription(result.description) }}</span>
+                </md-card-header>
+              </md-card-area>
+            </md-card-media-cover>
+          </router-link>
+        </md-card>
+      </div>
+      <pagination
+        :cpage="page"
+        :tpages="totalPages"
+        @go-to-page="loadItems($event)"
+      />
+    </div>
+  </div>
+</template>
+<script>
+import spinner from '@/components/Spinner'
+import pagination from '@/components/explorer/Pagination'
+import defaultImg from '@/assets/img/rdf_flyer.svg'
+import { mapGetters, mapActions } from 'vuex'
+
+export default {
+  name: 'viz-grid',
+  props: {
+    authenticated: {
+      type: Boolean,
+      require: true
+    },
+    instancetype: {
+      type: String,
+      require: true
+    }
+  },
+  data () {
+    return {
+      loading: true,
+      loadError: false,
+      otherArgs: null,
+      defaultImg
+    }
+  },
+  components: {
+    pagination,
+    spinner
+  },
+  computed: {
+    ...mapGetters('explorer/gallery', ['items', 'page', 'total', 'totalPages'])
+  },
+  methods: {
+    ...mapActions('explorer/gallery', ['loadItems']),
+    reduceDescription (args) {
+      const arr = args.split(' ')
+      arr.splice(15)
+      const arrSplice = arr.reduce((a, b) => `${a} ${b}`, '')
+      const res = arrSplice.normalize('NFD').replace(/[\u0300-\u036f]/g, '')
+      return `${res}...`
+    },
+    deleteChart (chart) {
+      console.log('delete chart', chart)
+    },
+    async loadItems (page = 1) {
+      this.loading = true
+      await this.$store.dispatch('explorer/gallery/loadItems', { page })
+      this.loading = false
+    }
+  },
+  async mounted () {
+    await this.loadItems()
+  }
+}
+</script>
+<style lang="scss" scoped>
+@import "@/assets/css/base/_grid.scss";
+@import "@/assets/css/abstract/_mixins.scss";
+@import "@/assets/css/modules/_utility.scss";
+.viz-content {
```

Comment:
Importing won't be needed here, since CSS is globally available to all components. If it doesn't work that way, we might need to look further into that with a separate ticket

Reply:
removed all but the mixins import, since there is a mixin used

Reply:
Thanks! I'll suggest merging the mixins into one file so we won't have to import that as well.